### PR TITLE
sound.ts added

### PR DESCRIPTION
**Pull request by Vidhanvyrs**

### CODE REVIEW

File: src/Components/Sound.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      this.audio.loop = true;
+    } else if (count > 0) {
+      this.audio.loop = false;
+      this.audio.addEventListener('ended', () => {
+        if (count > 1) {
+          count--;
+          this.audio.currentTime = 0;
+          this.audio.play();
+        }
+      });
```
Comment (corners2wall) [inline src/Components/Sound.ts]: Please move this function into private method  `decreaseCounter`

File: src/Components/Sound.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+const tapBlockSound = new Sound('public/sound/tapBlock.wav');
+const gameOverSound = new Sound('public/sound/gameOverSound.wav');
+
+// To play the sounds:
+tapBlockSound.play(); 
+gameOverSound.play(); 
+
+
+tapBlockSound.setVolume(0.5); 
+gameOverSound.repeat(2);
```
Comment (corners2wall) [inline src/Components/Sound.ts]: Please remove this

File: src/Components/Sound.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
+
+interface Sound {
+  play(): void; // play sound
+  stop(): void; // stop sound
+  repeat(count: number): void; // repeat sound `count` times or infinity
+  setVolume(value: number): void; // set volume of sound
+}
```
Comment (corners2wall) [inline src/Components/Sound.ts]: Please remove comments from interface 
Rename interface to `ISound`

File: src/Components/Sound.ts [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+class Sound implements Sound {
```
Comment (corners2wall) [inline src/Components/Sound.ts]: Change to 
```typescript
export default class Sound implements ISound {
```

File: src/Components/Sound.ts [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+  stop() {
+    this.audio.pause();
+    this.audio.currentTime = 0;
+  }
+
+  repeat(count: number) {
+    if (count === Infinity) {
+      this.audio.loop = true;
+    } else if (count > 0) {
```
Comment (corners2wall) [inline src/Components/Sound.ts:28]: Better: 
```typescript
if (count > 0) {
```

**Review by corners2wall — COMMENTED** [review]
Keep going

**Comment by Vidhanvyrs** [comment]
on it @corners2wall

**Comment by corners2wall** [comment]
@Vidhanvyrs  Good job!!!

**Comment by Vidhanvyrs** [comment]
> @Vidhanvyrs  Good job!!!

Thankyou 🙌

**Pull request merged**